component/admin/controllers/cpanel.php

```diff
@@ -53,6 +53,17 @@ function cpanel()
 		if ($rsvpproplugin && $rsvpplugin)
 		{
 			$app->enqueueMessage(Text::_("JEV_INSTALLED_RSVP_AND_RSVPPRO_INCOMPATIBLE"), "ERROR");
+			$db = Factory::getDbo();
+			$query = $db->getQuery(true)
+				->update('#__extensions')
+				->set('enabled = 0')
+				->where('enabled = 1')
+				->where('type = ' . $db->quote('plugin'))
+				->where('folder = ' . $db->quote('jevents'))
+				->where('element = ' . $db->quote('jevrsvp'))
+				->where('state IN (0,1)');
+			$db->setQuery($query);
+			$db->execute();
 		}
```

Comment:
I already made this change on 15th September - don't know why it doesn't appear in Git???

Reply:
Doh!  its from the merge but why does it appear here since its already in git???